src/base/stack-mixin.js

```diff
@@ -15,50 +16,46 @@ export class StackMixin extends CoordinateGridMixin {
 
         this._stackLayout = stack();
 
-        this._stack = [];
+        this._dataProvider = new StackedDataProvider();
```

Comment:
Why is the data provider reused by containment, while most other things are reused by inheritance?